tests/README.md

```diff
@@ -14,26 +14,51 @@ tests
 |    |____exec                                    ---- exec contains the different services to run the commands
 |    |    |____rosacli                            ---- ROSA CLI specific services/commands
 |    |____log                                     ---- tests logger
+|    |____profilehandler                          ---- package which support create cluster by profile configuration
 |       
 |____prow_ci.sh
 ```
 
 ## Contibute to ROSA CLI tests
 
 Please read the structure and contribute code to the correct place
+### Contribute to day1
+1. Enable the configuration in _[profiles](./ci/data/profiles)_
+2. Mapping the configuration in _[ClusterConfig](./utils/profilehandler/interface.go)_
+3. Define the userdata preparation function in _[data_preparation](./utils/profilehandler/data_preparation.go)_
+4. Call the functions in the _[GenerateClusterCreateFlags](./utils/profilehandler/profile_handler.go)_
 
 ### Contribute to day2
 
-* Create the case in rosa/tests/e2e/<feature name>_test.go
+* Create the case in _rosa/tests/e2e/{feature name}_test.go_
 * Label the case with ***CI.Day2***
 * Label the case with importance ***CI.Critical*** or ***CI.High***
 * Don't need to run creation step, just in BeforeEach step call function  ***config.GetClusterID()*** it will load the clusterID prepared from env or cluster_id file
 * Code for day2 actions and check step
 * Every case need to recover the cluster after the case run finished unless it's un-recoverable
+* Case format should follow 
+  * _main feature description in Describe level_ at the same time _testing purpose in It level_.
+  * `id` of the case is included which will follow the fmt of _[id:<id>]_
+  * Use `By("")` to describe the steps
+  * An example as below
+```golang
+var _ = Describe("Create Machine Pool", func() {
+  It("to hosted cluster with additional security group IDs will work [id:72195]",func(){
```

Comment:
It should be `work  - [id:72195]`

Reply:
BTW, the commit requirements are:
1. Only 1 commit per PR
2. Commit message should follow up https://github.com/openshift/rosa/blob/master/CONTRIBUTING.md#contributing-to-rosa with the automated/updated case ids in the format `id:case_id,case_id`.